manifests/n/NeoAxis/Engine/2024.1/NeoAxis.Engine.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: NeoAxis.Engine
+PackageVersion: '2024.1'
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-05-25
+AppsAndFeaturesEntries:
+  - DisplayName: NeoAxis Launcher version 2024.1
+    DisplayVersion: 2024.1

```

Comment:
```suggestion
```
DisplayVersion is not needed when `PackageVersion` matches exactly and there are no other manifests for the package which specify a DisplayVersion